npm-shrinkwrap.json

```diff
@@ -2,7 +2,7 @@
   "name": "marathon-ui",
   "version": "0.1.0",
   "npm-shrinkwrap-version": "5.4.0",
-  "node-version": "v0.10.37",
+  "node-version": "v1.8.2",
```

Comment:
Let's revert this and get in a separate PR with a conscious decision about which version to use. 0.12 is used in travis: https://github.com/mesosphere/marathon-ui/blob/master/.travis.yml#L7